.gitignore

```diff
@@ -0,0 +1 @@
+**/.DS_Store
```

Comment:
(you don't need the `**/` here, patterns are recursive unless you have a `/` in them)

Reply:
@BernhardValenti Fixed. Please review.